README.md

```diff
@@ -159,9 +159,9 @@ To run tests, run the following command:
 
 > Describe 1 - 3 features you will add to the project.
 
```

Comment:
## 1.
![image](https://user-images.githubusercontent.com/93039359/215242176-27924a94-d4cf-4b89-9741-d40076f9f96d.png)
## 2.
![image](https://user-images.githubusercontent.com/93039359/215242196-6f15bd94-9a7b-4eeb-8f0e-24ed994233e5.png)

## 3.
![image](https://user-images.githubusercontent.com/93039359/215242212-846630ac-1167-46c8-92f2-cd34a0b42f86.png)

## 4.
![image](https://user-images.githubusercontent.com/93039359/215242224-fc76fa9a-a927-498a-9ff7-99bb124316ec.png)

## 5.
![image](https://user-images.githubusercontent.com/93039359/215242234-22b287b9-c693-42da-b2fb-bcf2e707bfb1.png)
## 6.
![image](https://user-images.githubusercontent.com/93039359/215242247-a0ef736a-c7d0-4b0e-85e3-9f1245cde213.png)
## 7.
![image](https://user-images.githubusercontent.com/93039359/215242265-3f9e0546-f99c-46ca-be0c-a1d9a7df53a1.png)

## 8.
![image](https://user-images.githubusercontent.com/93039359/215242281-807262bb-a5a9-4eec-9392-778a24e28528.png)

## 9.
![image](https://user-images.githubusercontent.com/93039359/215242292-70b3240e-eb4b-407e-9577-0c862d421b25.png)

## 10.
![image](https://user-images.githubusercontent.com/93039359/215242297-4c34b5f2-c744-482c-bf01-044d19aa1bf0.png)

- [ ] Kindly remove all these sections descriptions and instructions which are not making your readme look clean and professional :blush: